src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -39,16 +49,6 @@ $poster-preview-width: 300px;
     flex-wrap: wrap;
   }
 
-  &__category {
-    display: flex;
-    font-size: $font-size--xs;
-    color: $white;
-    cursor: pointer;
-    padding: $spacing--sm;
-    margin: $spacing--xs;
-    border-radius: $border-radius--lg;
-  }
```

Comment:
Is this no longer needed here because `src/components/atoms/PosterCategory/PosterCategory.tsx` was extracted?

Reply:
Yes